dispatcher_plugin_nb2workflow/dataserver_dispatcher.py

```diff
@@ -98,6 +98,44 @@ def test_has_input_products(self, instrument, logger=None):
         query_out.set_done('input products check skipped')
         return query_out, []
 
+    def get_progress_run(self, **kwargs):
+
+        query_out = QueryOutput()
+        res_trace = None
+
+        task = kwargs.get('task', self.task)
+        param_dict = kwargs.get('param_dict', self.param_dict)
+
+        run_asynch = kwargs.get('run_asynch', None)
+        call_back_url = kwargs.get('call_back_url', None)
```

Comment:
Not sure if `_async_request_callback` arg is needed in this request

Reply:
let me know, I will remove it in case

Reply:
It's not used to calculate the hash, by which the async job is identified. 
So if the job is already submitted to the async job queue by nb2service backend, the plugin will get the response with the status and jobdir regardless of this parameter. 

Some edge cases came to mind, I will summarise in the separate message